spec/features/user_log_in_spec.rb

```diff
@@ -0,0 +1,15 @@
+feature 'User log in' do
+  let!(:user) do
+    User.create(email: 'user@example.com',
+                password: 'secret1234',
+                password_confirmation: 'secret1234',
+                name: 'user',
+                user_name: 'user1')
+  end
+
+  scenario 'with correct credentials' do
+    sign_in(email: user.email, password: user.password)
+    expect(page).to have_content "Welcome, #{user.email}"
+  end
```

Comment:
good sign_in method - the argument makes it really clear that you should be seeing the users email.